ee/codegen/src/generators/node-display.ts

```diff
@@ -4,6 +4,7 @@ import { Writer } from "@fern-api/python-ast/core/Writer";
 import { isNil } from "lodash";
 
 import { BaseNodeContext } from "src/context/node-context/base";
```

Comment:
We should add a test for generating base node's where it has the default color + icon in its display data and we verified that it was skipped